assignments/this.js

```diff
@@ -1,26 +1,67 @@
-/* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+//  The for principles of "this";
+//  in your own words. explain the four principle for the "this" keyword below.
+
+//   1. window binding occurs when no other binding has been assigned and "this" will be bound to the global object
+// * 2. implicit is binding in object methods with this reffering to the object before the dot.
+// * 3.  new keyword binding creates a new object and binds it to this
+// * 4. explicit occurs when .call is used on a function and you are passing explicitly in a this context
+// *
+// * write out a code example of each explanation above
+// 
 
 // Principle 1
 
 // code example for Window Binding
-
+function hello(){
+    "use strict"
```

Comment:
unless you want it only to this function, having it at the top of the file is better.